R/data.table.R

```diff
@@ -1666,13 +1667,12 @@ replace_order = function(isub, verbose, env) {
       nomeanopt=FALSE  # to be set by .optmean() using <<- inside it
       oldjsub = jsub
       if (jsub[[1L]]=="list") {
-        for (ii in seq_along(jsub)[-1L]) {
-          this_jsub = jsub[[ii]]
-          if (dotN(this_jsub)) next; # For #5760
```

Comment:
this `dotN` thing isn't doing anything? since this loop only affects `is.call` elements & `dotN` specifically checks `is.name`.